### DISCUSSION TITLE
Better write Deno.js

### DISCUSSION THREAD

**Discussion by danny007in** [lapce/lapce#477] [Polls]
https://deno.land/

Deno is a simple, modern and secure runtime for JavaScript, TypeScript, and WebAssembly that uses V8 and is built in Rust.

**Comment by Kyllingene** [comment]
What is the poll actually about? What does "implementing" Deno in Lapce mean?

**Reply by danny007in** [reply]
deno is anyhow written in Rust, so we can directly use deno with extra features

**Comment by setop** [comment]
Why not Lua or Python as embedded scripting languages also ?
